packages/next/src/build/collect-build-traces.ts

```diff
@@ -463,6 +471,19 @@ export async function collectBuildTraces({
               curTracedFiles.add(file)
             }
 
+            if (process.env.NEXT_RSPACK) {
+              for (const file of chunks) {
```

Comment:
not sure whether these changes needed since it seems TraceEntryPointsPlugin runs on Rspack